project-fortis-interfaces/src/components/Insights/Maps/HeatMap.js

```diff
@@ -75,10 +75,14 @@ export default class HeatMap extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     const { placeid, defaultZoom } = this.state;
+    const { targetBbox } = this.props;
 
     if (hasChanged(this.props, nextProps) && nextProps.selectedplace.placeid && placeid !== nextProps.selectedplace.placeid) {
       this.moveMapToNewLocation(nextProps, defaultZoom);
     }
+    if (hasChanged(this.props, nextProps) && nextProps.bbox && JSON.stringify(nextProps.bbox) === JSON.stringify(targetBbox)) {
```

Comment:
You can use lodash's [isEqual](https://lodash.com/docs/4.17.5#isEqual) for this instead of relying on the string representation:

```js
import isEqual from 'lodash/isEqual';
isEqual(nextProps.bbox, targetBbox);
```